pandas/core/generic.py

```diff
@@ -1786,40 +1786,47 @@ def to_json(self, path_or_buf=None, orient=None, date_format=None,
                             index=index)
 
     def to_hdf(self, path_or_buf, key, **kwargs):
-        """Write the contained data to an HDF5 file using HDFStore.
+        """
+        Write the contained data to an HDF5 file using HDFStore.
+
+        Hierarchical Data Format (HDF) is self-describing, allowing an
+        application to interpret the structure and contents of a file with
+        no outside information. One HDF file can hold a mix of related objects
+        which can be accessed as a group or as individual objects.
+
+        In order to add another :class:`~pandas.DataFrame` or
+        :class:`~pandas.Series` to an existing HDF file please use append mode
+        and different a key.
 
```

Comment:
Can you add here a link to the user guide (because there is a lot more information there). You can use something like `` For more information see the :ref:`user guide <io.hdf5>`  ``

Reply:
Thanks for the hint! This one is hard for me, because i'm not an expert in rst :( I put the line  as you suggested, with the right subsection of that manual. when i generate html with 'make.py html' it doesnt' have a link. Is it ok?

Reply:
Yes, that is fine (when only building the docstring, the full user guide is not built, and therefore the link does not seem to work)